src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -62,8 +73,8 @@ struct cbuf_maps {
 };
 
 struct cbuf_info {
-	u32_t cbid;
-	int size;
+	int cbid;
```

Comment:
Why changing cbid to a signed representation? Do we use negative cbids somewhere? I believe the cbid is really a 30-bit unsigned number.